source/extensions/access_loggers/grpc/http_grpc_access_log_impl.cc

```diff
@@ -139,11 +142,13 @@ void HttpGrpcAccessLog::emitLog(const Http::RequestHeaderMap& request_headers,
     auto* logged_headers = response_properties->mutable_response_headers();
 
     for (const auto& header : response_headers_to_log_) {
+      // TODO(https://github.com/envoyproxy/envoy/issues/13454): Potentially log all header
+      // values.
       const auto entry = response_headers.get(header);
       if (!entry.empty()) {
-        // TODO(https://github.com/envoyproxy/envoy/issues/13454): Potentially log all header
-        // values.
-        logged_headers->insert({header.get(), std::string(entry[0]->value().getStringView())});
+        const auto all_values = Http::HeaderUtility::getAllOfHeaderAsString(
+            response_headers, Http::LowerCaseString(header));
```

Comment:
It seems that it is unnecessary to convert `header` to `Http::LowerCaseString` with `Http::LowerCaseString(header)`. The type of header was originally  `Http::LowerCaseString`.